### PR TITLE
Kh pylypets/registration component ui

### PR DESCRIPTION
**Pull request by khrystynapylypets**

### CODE REVIEW

**Review by InnaKutsik — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -107,9 +107,11 @@
     "ramda": "^0.26.1",
     "react": "^16.9.0",
     "react-bem-helper": "^1.4.1",
+    "react-bootstrap": "^1.0.0-beta.11",
```
Comment (anna-kuzii) [inline package.json:110]: stable version

File: src/app/components/Registration/Registration.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+export const Registration = () => (
+
+  <Container className='registrationWrapper'>
+    <h1>Create Account</h1>
+    <section className='socialIcons'>
+      <SVG src={facebookIcon} />
+      <SVG src={twitterIcon} />
+    </section>
+    <form method='post' className='formWrapper'>
```
Comment (anna-kuzii) [inline src/app/components/Registration/Registration.js]: remove higher to containrer

File: src/app/components/Registration/registration.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      height: 32px;
+
+      &:first-of-type {
+        margin-right: 10px;
+      }
+    }
+  }
+}
+
+.formWrapper {
```
Comment (anna-kuzii) [inline src/app/components/Registration/registration.scss]: move to forms.scss

File: src/styles/general.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@import 'vars';
+
+body,
+html {
+  padding: 0;
+  margin: 0;
+  font-family: $roboto-regular, sans-serif;
+  box-sizing: border-box;
```
Comment (anna-kuzii) [inline src/styles/general.scss]: inherit? if not -- use *

File: src/styles/vars.scss [hunk 5]
```diff
@@ -1,10 +1,22 @@
-$font-color: #444;
+$roboto-regular: 'Roboto Regular';
+$roboto-bold: 'Roboto Bold';
+$sans-regular: 'Open Sans';
+$sans-bold: 'Open Sans Bold';
 
+$font-color: #444;
 $info-blue: #0e8dce;
 $error-red: #af0000;
```
Comment (anna-kuzii) [inline src/styles/vars.scss]: rm colors

**Review by anna-kuzii — CHANGES_REQUESTED** [review]

File: src/app/components/Registration/registration.scss [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        margin-right: 10px;
+      }
+
+      &:hover {
+        path {
+          fill: $alizarin-crimson;
+        }
+      }
+
+      &.facebookIcon {
```
Comment (anna-kuzii) [inline src/app/components/Registration/registration.scss]: make smaller nesting depth

File: src/app/components/Registration/Registration.js [hunk 7]
```diff
@@ -0,0 +1,84 @@
+import React from 'react'
```
Comment (anna-kuzii) [inline src/app/components/Registration/Registration.js]: rename file to index

File: src/app/components/Registration/registration.scss [hunk 8]
```diff
@@ -0,0 +1,51 @@
+@import '../../../styles/vars';
```
Comment (anna-kuzii) [inline src/app/components/Registration/registration.scss]: rename file to styles

File: src/app/components/Registration/assets/facebook-icon.svg [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<svg width="34" height="34" viewBox="0 0 34 34" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (anna-kuzii) [inline src/app/components/Registration/assets/facebook-icon.svg:1]: delete unnecessary imgs from this folder, pls

File: src/app/components/Registration/Registration.js (hunk 2 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/Registration.js]: done

File: src/app/components/Registration/registration.scss (hunk 3 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/registration.scss]: done

File: src/styles/general.scss (hunk 4 above)
Comment (khrystynapylypets) [reply src/styles/general.scss]: fixed, used *

File: src/styles/vars.scss (hunk 5 above)
Comment (khrystynapylypets) [reply src/styles/vars.scss]: removed

File: src/app/components/Registration/registration.scss (hunk 6 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/registration.scss]: fixed

File: src/app/components/Registration/Registration.js (hunk 7 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/Registration.js]: renamed

File: src/app/components/Registration/registration.scss (hunk 8 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/registration.scss]: renamed

File: src/app/components/Registration/assets/facebook-icon.svg (hunk 9 above)
Comment (khrystynapylypets) [reply src/app/components/Registration/assets/facebook-icon.svg:1]: fixed

**Review by anna-kuzii — APPROVED** [review]

**Review by InnaKutsik — APPROVED** [review]

**Pull request merged**